pages/team.tsx

```diff
@@ -4,15 +4,18 @@ import { Trans } from 'react-i18next';
 
 import { useI18n } from '@packages/features/i18n-context';
 import Section from '@packages/components/Section';
+import MemberCardProfile from '@packages/components/MemberCard';
```

Comment:
O nome do componente idealmente tem que bater com o nome do import. Melhor alterar o import
```suggestion
import MemberCard from '@packages/components/MemberCard';
```